src/main/webapp/i18n/de/assessment.json

```
@@ -64,7 +64,7 @@
                 "warning": "Achtung: Die Einreichungsfrist ist noch nicht abgelaufen und Studierende dürfen noch ihre Einreichungen abgeben. Bitte noch nicht mit der Korrektur anfangen.",
                 "warningIndividual": "Achtung: Die allgemeine Einreichungsfrist ist abgelaufen, aber manche Studierende mit verlängerter Bearbeitungszeit dürfen noch ihre Einreichungen abgeben. Bitte stelle vor Beginn einer Korrektur sicher, dass diese Studierenden nicht mehr abgeben können.",
                 "empty": "Keine Abgabe gefunden!",
-                "emptySearch": "Bitte gib einen Studentennamen ein, um nach Abgaben zu suchen.",
+                "emptySearch": "Bitte gib einen Studierendennamen ein, um nach Abgaben zu suchen.",
```

```suggestion
                "emptySearch": "Bitte gib den Namen eines Studierenden ein, um nach Abgaben zu suchen.",
```
Weird neologism/compound word imo